src/hesian.f90

```diff
@@ -90,6 +92,11 @@ subroutine hesian( NGdof, position, Mvol, mn, LGdof )
 
   BEGIN(hesian)
 
+  ! Only makes sense to compute the Hessian if helicity is constrained
```

Comment:
Is this just a check or could the code actually arrive here? In that case it is better if the user is given a warning that they have their configuration wrong.